src/Nethermind/Nethermind.Network.Stats/Model/DisconnectReason.cs

```diff
@@ -1,25 +1,149 @@
 // SPDX-FileCopyrightText: 2022 Demerzel Solutions Limited
 // SPDX-License-Identifier: LGPL-3.0-only
 
-namespace Nethermind.Stats.Model
+namespace Nethermind.Stats.Model;
+
+/// <summary>
+/// Nethermind level disconnect reason. Don't forget to add the corresponding Eth level disconnect reason in `InitiateDisconnectReasonExtension`.
+/// </summary>
+public enum DisconnectReason : byte
```

Comment:
Can you add more info why Nethermind DisconnectReason is there with EthDisconnectReason?

Reply:
Done